### ISSUE TITLE
feature request: show output in 2 columns

### ISSUE THREAD

**Issue by bbigras** [afnanenayet/diffsitter#158]
Any plans to make the output in 2 columns like delta and difftastic?

In this example, I find the difftastic's version really pretty.

delta:
![image](https://user-images.githubusercontent.com/24027/126273375-748d0b99-71d9-42c5-97e6-4ca7a87bb84f.png)

diffsitter:
![image](https://user-images.githubusercontent.com/24027/126273441-0d5559fc-0838-446f-83b4-06975ce11be2.png)

difftastic:
![image](https://user-images.githubusercontent.com/24027/126273541-658c5ec3-0ce3-4ecf-b6d7-2026dec36bfe.png)

**Comment by afnanenayet** [comment]
Yeah this would be cool, I like the split view a lot. It would probably be good to integrate this thing with a pager too.

**Comment by waldyrious** [comment]
Interestingly, there's a request for precisely the opposite feature in difftastic :) https://github.com/Wilfred/difftastic/issues/34
